geopyspark/vector_pipe/protobuf/featureMessages_pb2.py

```diff
@@ -1,352 +0,0 @@
-# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Are these potentially useful outside of interface with vectorpipe ?

Reply:
I don't think so. That code is specifically for an older version of `featureMessages.proto`.